### ISSUE TITLE
Currently broken with Fable

### ISSUE THREAD

**Issue by njlr** [fsprojects/FSharp.Control.AsyncSeq#145]
### Description

Latest Nuget package (`3.1.0`) is broken with Fable. 

It appears that Fable tests are not part of the CI, probably why it slipped. 

I have tried building the code, but was unable to get things working. 

Some steps:

```bash
$ git --no-pager log -n 1
commit 4474df189a512e3f129ef9482dd19a7da8cf71a3 (HEAD -> master, origin/master)
Author: Don Syme <donsyme@fastmail.com>
Date:   Sat Jul 24 17:12:51 2021 +0100

    fix publish

$ dotnet tool install fable
$ dotnet fable ./src/FSharp.Control.AsyncSeq/

...

FSharp.Control.AsyncSeq/src/FSharp.Control.AsyncSeq/AsyncSeq.fs(1,1): error EXCEPTION: Cannot find dll entity System.Collections.Generic.IEnumerable`1
Compilation failed
```

### Related information 

* Operating system Ubuntu `20.04`
* Branch `master`
* Database versions and sample databases being used: None
* .NET Runtime, CoreCLR or Mono Version: Fable `3.2.14`
* Performance information, links to performance testing scripts: None

**Comment by dsyme** [comment]
@alfonsogarciacaro Could you or someone else familiar with debugging Fable failures take a look please?  Thanks!

**Comment by alfonsogarciacaro** [comment]
@njlr So you mean the previous Nuget package was working fine with Fable? The error is weird, IEnumerable type should be easily found. I will try to debug.

**Comment by njlr** [comment]
> @njlr So you mean the previous Nuget package was working fine with Fable? The error is weird, IEnumerable type should be easily found. I will try to debug.

Yes, a previous version was working with Fable (sorry can't recall which it was, but it was several versions ago). I tried to build the latest code with Fable 3 but hit this error.

**Comment by dsyme** [comment]
@alfonsogarciacaro @njlr I'm not aware of how to get this working with Fable latest - please advise?

Thanks

**Comment by alfonsogarciacaro** [comment]
@dsyme Sorry, forgot about this one! I've sent a new PR to update the tests #148. With that Fable tests will also run in CI (at least for PRs). I hope it also helps with this issue although I couldn't reproduce the exact problem reported by @njlr.

**Comment by njlr** [comment]
https://github.com/fsprojects/FSharp.Control.AsyncSeq/pull/149
